src/index.native.tsx

```diff
@@ -379,6 +408,64 @@ const ScreenStackHeaderBackButtonImage = (props: ImageProps): JSX.Element => (
   </ScreensNativeModules.NativeScreenStackHeaderSubview>
 );
 
+class SearchBar extends React.Component<SearchBarProps> {
+  nativeSearchBarRef: React.RefObject<SearchBarCommands>;
+
+  constructor(props: SearchBarProps) {
+    super(props);
+    this.nativeSearchBarRef = React.createRef();
+  }
+
+  blur() {
+    if (this.nativeSearchBarRef.current) {
```

Comment:
Maybe it would be good to warn the user if the ref is not yet set so it is known why it did not work?

Reply:
[752244b](https://github.com/software-mansion/react-native-screens/pull/1610/commits/752244b85f28cfe79a8a4542a89aeb7d3be506dd)